miniwallet/deploy/003_deploy_mini721.ts

```diff
@@ -0,0 +1,92 @@
+import config from '../config'
+import { HardhatRuntimeEnvironment } from 'hardhat/types'
+import { DeployFunction } from 'hardhat-deploy/types'
+import { ethers } from 'hardhat'
+
+const deployFunction: DeployFunction = async function (
+  hre: HardhatRuntimeEnvironment
+) {
+  const { deployments, getNamedAccounts, getChainId } = hre
+  const { deploy } = deployments
+  const { deployer, operatorA } = await getNamedAccounts()
+  const chainId = await getChainId()
+  let mini721DeployArgs = {}
+  let saleIsActive
+  let metadataFrozen
+  let provenanceFrozen
+  let max721Tokens
+  let mintPrice
+  let maxPerMint
+  let baseUri
+  let contractUri
+
+  console.log(`chainId: ${chainId}`)
+  if (chainId === '1666600000,') {
+    console.log('Harmony Mainnet Deploy')
+    console.log(`Min721 Deployment Info: ${JSON.stringify(config.mainnet.mini721)}`)
+    saleIsActive = config.mainnet.mini721.saleIsActive
+    metadataFrozen = config.mainnet.mini721.metadataFrozen
+    provenanceFrozen = config.mainnet.mini721.provenanceFrozen
+    max721Tokens = config.mainnet.mini721.max721Tokens
+    mintPrice = config.mainnet.mini721.mintPrice
+    maxPerMint = config.mainnet.mini721.maxPerMint
+    baseUri = config.mainnet.mini721.baseUri
+    contractUri = config.mainnet.mini721.contractUri
+  } else {
+    console.log(`Test Deploy on chainId: ${chainId}`)
+    console.log(`Min721 Deployment Info: ${JSON.stringify(config.test.mini721)}`)
+    mini721DeployArgs = config.test.mini721
+    saleIsActive = config.test.mini721.saleIsActive
+    metadataFrozen = config.test.mini721.metadataFrozen
+    provenanceFrozen = config.test.mini721.provenanceFrozen
+    max721Tokens = config.test.mini721.max721Tokens
+    mintPrice = config.test.mini721.mintPrice
+    maxPerMint = config.test.mini721.maxPerMint
+    baseUri = config.test.mini721.baseUri
+    contractUri = config.test.mini721.contractUri
+  }
+  console.log('mini721DeployArgs:', JSON.stringify(mini721DeployArgs))
+
+  const deployedMini721 = await deploy('Mini721', {
+    from: deployer,
+    gasLimit: 4000000,
```

Comment:
why?

Reply:
Removed 🙏 
Was initially put in as was getting unexpected gas limit, but this was due to errors in deploy logic